src/mpid/ch4/shm/posix/eager/iqueue/iqueue_init.c

```diff
@@ -31,6 +31,16 @@
       description : >-
         Size of each cell.
 
+    - name        : MPIR_CVAR_CH4_SHM_POSIX_IQUEUE_FREE_QUEUE_SENDER_SIDE
+      category    : CH4
+      type        : boolean
+      default     : true
+      class       : none
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        Control if sender side or receiver side free cell queue is used.
+
```

Comment:
Don't move CVAR in this commit since you are going to delete it the next commit anyway.

Reply:
WIll fix.